cmd/kots/cli/install.go

```diff
@@ -444,11 +448,22 @@ func InstallCmd() *cobra.Command {
 				switch status {
 				case storetypes.VersionPendingPreflight:
 					log.ActionWithSpinner("Waiting for preflight checks to complete")
+					log.FinishSpinner()
 					if err := ValidatePreflightStatus(deployOptions, authSlug, apiEndpoint); err != nil {
-						log.FinishSpinnerWithError()
-						return errors.Wrap(err, "failed to validate preflight results")
+						log.Errorf("Preflight checks contain warnings or errors. The application was not deployed")
+						perr := preflightError{}
+						if errors.As(err, &perr) {
+							cmd.SilenceErrors = true
```

Comment:
Do we need to silence errors if we're always going to call `os.Exit(1)` at the end of this code path?

Reply:
> Do we need to silence errors if we're always going to call `os.Exit(1)` at the end of this code path?

Perhaps not. Removing it